### ISSUE TITLE
Detect class in argument type hinting

### ISSUE THREAD

**Issue by mauris** [packfire-archived/pdc#10]
PDC doesn't detect argument type hinting and hence declares `use` statement to be unused.

**Comment by mauris** [comment]
Codes like:

```
/**
 * Packfire Dependency Checker (pdc)
 * By Sam-Mauris Yong
 * 
 * Released open source under New BSD 3-Clause License.
 * Copyright (c) Sam-Mauris Yong Shan Xian <sam@mauris.sg>
 * All rights reserved.
 */

namespace Packfire\PDC\Analyzer;

use Packfire\PDC\Report\IReport;

interface IAnalyzer {

    public function __construct($file);

    public function analyze(IReport $report);

}
```

will cause IReport to fail the unused check because the method signature does not register the use.

**Comment by mauris** [comment]
fixed in ba95ccc!
